pkg/api/types.go

```diff
@@ -1850,6 +1854,27 @@ func (p *Properties) SetCloudProviderRateLimitDefaults() {
 			p.OrchestratorProfile.KubernetesConfig.CloudProviderRateLimitQPS = DefaultKubernetesCloudProviderRateLimitQPS
 		}
 	}
```

Comment:
@feiskyer, I was hoping you could verify the default rate limit settings for bucket and qps write. Do the values seem reasonable or should they be decreased?

Reply:
the values look good to me.